### PR TITLE
fix(test): Fix trends test timeout

### PR DESCRIPTION
**Pull request by k-fish**
### Summary
Occasionally during pre-commit the trends test will timeout, going to fix tick/update for these tests to see if it happens, will open up another ticket later if it is still not fixed

### CODE REVIEW

File: tests/js/spec/views/performance/trends.spec.jsx [hunk 1]
```diff
@@ -370,8 +370,10 @@ describe('Performance > Trends', function () {
     );
 
     for (const trendFunction of TRENDS_FUNCTIONS) {
-      selectTrendFunction(wrapper, trendFunction.field);
       await tick();
+      wrapper.update();
+
+      selectTrendFunction(wrapper, trendFunction.field);
```
Comment (Zylphrex) [inline tests/js/spec/views/performance/trends.spec.jsx:376]: Should this be before the tick + update?

File: tests/js/spec/views/performance/trends.spec.jsx [hunk 2]
```diff
@@ -395,8 +397,10 @@ describe('Performance > Trends', function () {
     );
 
     for (const confidenceLevel of CONFIDENCE_LEVELS) {
-      selectConfidenceLevel(wrapper, confidenceLevel.label);
       await tick();
+      wrapper.update();
+
+      selectConfidenceLevel(wrapper, confidenceLevel.label);
```
Comment (Zylphrex) [inline tests/js/spec/views/performance/trends.spec.jsx:403]: Same with this one.

File: tests/js/spec/views/performance/trends.spec.jsx (hunk 1 above)
Comment (k-fish) [reply tests/js/spec/views/performance/trends.spec.jsx:376]: Actually I should do both to be safe, I had the one above for first run...

File: tests/js/spec/views/performance/trends.spec.jsx (hunk 2 above)
Comment (k-fish) [reply tests/js/spec/views/performance/trends.spec.jsx:403]: Ditto

**Review by Zylphrex — APPROVED** [review]

**Pull request merged**